test/tools/yulopti.cpp

```diff
@@ -214,47 +341,50 @@ class YulOpti
 					case '#':
 						return;
 					case ',':
-						VarNameCleaner::run(m_context, *m_ast);
+						runVarNameCleaner();
 						// VarNameCleaner destroys the unique names guarantee of the disambiguator.
 						disambiguated = false;
 						break;
 					case ';':
 					{
-						Object obj;
-						obj.code = m_ast;
-						StackCompressor::run(m_dialect, obj, true, 16);
+						runStackCompressor();
 						break;
 					}
 					default:
-						OptimiserSuite{m_context}.runSequence(
-							std::string_view(&option, 1),
-							*m_ast
-						);
+						runSequence(std::string_view(&option, 1));
```

Comment:
```suggestion
						runSequence(string_view(&option, 1));
```
We are already using namespace `std`.

Reply:
Fixed